deploy/casper-kube/values.yaml

```diff
@@ -0,0 +1,105 @@
+# The current architectural approach does not support more than
+replicaCount: 5
+
+image:
+  repository: 878804750492.dkr.ecr.us-east-2.amazonaws.com/casper-kube-node
+  pullPolicy: Always
+  tag: latest
```

Comment:
Currently the `latest` image needs to be built and overriden to be compatible with the changes in this PR. this is not done yet. Another open user story is autmation of docker builds on merge to master to avoid having to do this manually.